index.html

```diff
@@ -1,20 +1,450 @@
+                    message, and return a promise [=reject|rejected=] with a new
+                    <code>OverconstrainedError</code> created by calling
+                    <code>OverconstrainedError(<var>failedConstraint</var>,
+                    <var>message</var>)</code>.
+                </li>
+              </ol>
+            </li>
+            <li>
+              <p>Let <var>requestedMediaTypes</var> be the set of media
+                types in <var>constraints</var> with either a dictionary
+                value or a value of <code>true</code>.</p>
+            </li>
+            <li>
+              <p>If the <a>current settings object</a>'s [=relevant global object=]'s
+                [=associated `Document`=] is NOT [=Document/fully active=] or does NOT
+                <a data-cite="!HTML/#gains-focus">have focus</a>, return a promise
+                [=reject|rejected=] with a {{DOMException}} object whose {{DOMException/name}}
+                attribute has the value {{InvalidStateError}}.
+              </p>
+            </li>
+            <li>
+              <p>Let <var>p</var> be a new promise.</p>
+            </li>
+            <li>
+              <p>Run the following steps in parallel:</p>
+              <ol>
+                <li>
+                  <p>For each media type <var>T</var> in
+                    <var>requestedMediaTypes</var>,
+                  </p>
+                  <ol>
+                    <li>
+                      <!-- Remark: Is getDisplayMedia currently violating this part of the spec? -->
+                      <p>If no sources of type <var>T</var> are available,
+                        <a>reject</a> <var>p</var> with a new
+                        {{DOMException}} object whose
+                        {{DOMException/name}} attribute has the value
+                        {{NotFoundError}}.
+                      </p>
+                    </li>
+                    <li>
+                      <p>Read the current [= permission state=] for obtaining
+                        sources of type <var>T</var> in the current browsing
+                        context. If the permission state is {{PermissionState/"denied"}}, jump to
+                        the step labeled <em>PermissionFailure</em> below.</p>
+                    </li>
+                  </ol>
+                </li>
+                <li>
+                  <p>Optionally, e.g., based on a previously-established
+                    user preference, for security reasons, or due to platform
+                    limitations, jump to the step labeled <em>Permission Failure</em> below.</p>
+                </li>
+                <li>
+                  <p>[=Prompt the user to choose=]
+                    one or more <a data-cite="screen-capture#dfn-display-surface">display surfaces</a>,
+                    for one or more {{PermissionDescriptor}} with its {{PermissionDescriptor/name}} set
```

Comment:
Why more than one `{{PermissionDescriptor}}`? One should be enough no?

Reply:
Indeed. Done in the next iteration.